src/sysjson.go

```diff
@@ -39,9 +50,13 @@ func main() {
 
 func statsHandler(w http.ResponseWriter, r *http.Request) {
 	resp := map[string]interface{}{}
-
-	loadModules(resp, r.URL.Query().Get("modules"))
-
+	
+	if len(*from) == 0 || (len(*from) > 0 && strings.HasPrefix(r.RemoteAddr, *from)) {
```

Comment:
What if we used `net.ParseCIDR` instead? It returns a IPNet type that has a `Contains()` method that seems to do what I describe above.